packages/opentelemetry/lib/tracing/serverInstrumenation.js

```diff
@@ -0,0 +1,81 @@
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
+const { ExpressInstrumentation } = require('@opentelemetry/instrumentation-express');
+
+class ServerInstrumentation {
+  constructor () {
+    this._httpInstrumentation = new HttpInstrumentation();
+    this._expressInstrumentation = new ExpressInstrumentation();
+    this._httpCurrentConfig = {
+      ignoreIncomingPaths: [],
+      ignoreOutgoingUrls: [],
+      applyCustomAttributesOnSpan: {}
+    };
+  }
+
+  updateCurrentConfig () {
+    this._httpInstrumentation.setConfig(this._httpCurrentConfig);
+  }
+
+  getInstrumentations () {
```

Comment:
we could use getters here (https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Functions/get)